x-pack/plugins/ingest_manager/server/saved_objects/index.ts

```diff
@@ -26,6 +26,7 @@ import {
   migrateSettingsToV7100,
   migrateAgentActionToV7100,
 } from './migrations/to_v7_10_0';
+import { migratePackagePolicyToV7110 } from './migrations/to_v7_11_0';
```

Comment:
Can this reside and be run outside Fleet? e.g. inside security?

It seems limited to Endpoint and I'd like to limit how much Fleet "knows" about the specifics of a package

Reply:
we could store store this file outside of Fleet, but doesn't the SO migration have to run as a part of the SO mapping definition?  Or could we import a mapping in security and then add an additional migration?